### PR TITLE
Base: Add icons to man pages for GUI applications

### PR DESCRIPTION
**Pull request by electrikmilk**
Differentiate terminal and GUI applications with icons next to the name of the application within the man pages of the Help application. I removed the short descriptions after the name of the application as they all seemed to be contained within the description and I feel it helps differentiate them better.

As suggested when @jntrnr revisited Serenity: https://youtu.be/_QAsHkEKvN0?t=480

Previous PR: https://github.com/SerenityOS/serenity/pull/11557

Screen Shot 2022-01-13 at 20 51 31
Screen Shot 2022-01-13 at 20 51 48

### CODE REVIEW

**Comment by electrikmilk** [comment]
I get the same issue in this PR with this branch. The failing check says...
```
You may need to run ./Meta/export-argsparser-manpages.sh on your system and commit/squash the resulting changes.
108
Error: Process completed with exit code 1.
```
and this happens...
```console
serenity % ./Meta/export-argsparser-manpages.sh
This script assumes passwordless sudo.
Password:
SERENITY_ARCH not given. Assuming i686.
BUILD_DIR not given. Assuming Build/i686.
Directory 'fsmount' already exists.
Manual cleanup needed. You might also need to unmount first.
```
What do I need to unmount? What manual cleanup do I need to do? Does this have to do with the images possibly?

File: Base/usr/share/man/man1/Profiler.md [hunk 1]
```diff
@@ -50,4 +50,3 @@ $ Profiler perfcore.123
 ## See also
 
 * [`perfcore`(5)](help://man/5/perfcore)
-
```
Comment (kleinesfilmroellchen) [inline Base/usr/share/man/man1/Profiler.md:53]: Why did you remove this trailing newline as well as the others?

Comment (electrikmilk) [reply Base/usr/share/man/man1/Profiler.md:53]: I tried not to with both of the editors I use, I'm not sure. I'll try with a basic text editor maybe.
